src/commands/weather.js

```diff
@@ -74,23 +77,25 @@ class Weather extends Aquarius.Command {
   weatherToEmoji(code) {
     switch (code) {
       case '0':    // Tornado
+        return EMOJI_LIST.TORNADO;
       case '1':    // Tropical Storm
       case '2':    // Hurricane
-        return EMOJI_LIST.TORNADO;
+        return EMOJI_LIST.HURRICANE;
       case '3':    // Severe Thunderstorms
       case '4':    // Thunderstorms
       case '37':   // Isolated Thunderstorms
       case '38':   // Scattered Thunderstorms
       case '39':   // Scattered Thunderstorms
       case '45':   // Thundershowers
-      case '47':   // Isolated Thundershowers
         return EMOJI_LIST.THUNDER;
+      case '47':   // Isolated Thundershowers
```

Comment:
If I look it again now, maybe not. But does it need to have clouds to symbolize snow?